website/static/overrides.css

```diff
@@ -1,5 +1,5 @@
-body, a, p, span, h1, h2, h3, h4, h5, li {
-  font-family: "Apple SD Gothic Neo", "AppleSDGothicNeo",  -apple-system, BlinkMacSystemFont, Helvetica, Arial, sans-serif;
+body {
+  font-family: "Apple SD Gothic Neo", "AppleSDGothicNeo",  -apple-system, BlinkMacSystemFont, Helvetica, Arial, sans-serif !important;
```

Comment:
Can you avoid important and instead use a more specific selector like `html body`?

Reply:
Unfortunately `html body` isn't specific enough to override the default CSS 😢